app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -11,6 +11,10 @@ module AuthnJwt
     IDENTITY_NOT_RETRIEVED_YET = "Identity not retrieved yet"
     URL_IDENTITY_PROVIDER_INTERFACE_NAME = "url-identity-provider"
     TOKEN_IDENTITY_PROVIDER_INTERFACE_NAME = "token-identity-provider"
+    IDENTITY_PATH_RESOURCE_NAME = "identity-path"
+    IDENTITY_PATH_DEFAULT_VALUE = ""
+    IDENTITY_PATH_CHARACTER_DELIMITER = "/"
```

Comment:
Is there another const that defines the character for all Conjur globally?

Reply:
No, we have consts files only for authenticators